### ISSUE TITLE
disable minimal share for certain nodes

### ISSUE THREAD

**Issue by yannickoo** [yannickoo/minimal_share#3]
We should implement some kind of configuration for the editor to disable the sharing links, this can be very useful for an _Imprint_ or _Terms_ page. This feature was requested in #1.

**Comment by glassdimly** [comment]
I agree here. I turned on Minimal Share and am quite pleased with it. However, I wanted to display as a block just on nodes. So I have to go through display settings for my 5 other content types, times 3 custom displays each and disable minimal share on each screen.

**Comment by yannickoo** [comment]
The problem you are reporting sounds more like the initial configuration. This is caused by Drupal's [`hook_field_extra_fields`](https://api.drupal.org/api/drupal/modules!field!field.api.php/function/hook_field_extra_fields/7.x) because you cannot set them to hidden. This has been fixed in the Drupal 8 version, see [`hook_entity_extra_field_info`](https://api.drupal.org/api/drupal/core!lib!Drupal!Core!Entity!entity.api.php/function/hook_entity_extra_field_info/8.2.x).

This behavior will be fixed in the `8.x` version of this module :)

**Comment by glassdimly** [comment]
Right. I wish that enabling them as files with hook_field_extra_fields were optional, or if there were a post-install hook that set all of them to "hidden." 

I would doubt that many people desire for Minimal Share to appear on every display type including teaser displays.

I guess I should make you a patch, no? hahaha. Maybe I will, but my time is pretty tight right now.
